docker/superset/public_welcome.html

```diff
@@ -26,6 +26,9 @@ <h1 class="display-4">Welcome to EpiGraphHub!</h1>
     <p>We are here to help you make sense of Public Health data!</p>
     <p class="lead">If you just want to browse the published dashboards and charts,
       please login with the <b>guest</b> account! password: <i>guest</i>.</p>
+    <p>If you already have a user account, click the <i>Login</i> button below, and if you want to create an account, click on the <i>Sign up</i> button below.</p>
+    <a href="{{appbuilder.get_url_for_login}}" role="button" class="btn btn-primary btn-lg">Login</a>
+    <a href="/register/form" role="button" class="btn btn-success btn-lg">Sign up!</a>
```

Comment:
```suggestion
    <a href="/register/form" role="button" class="btn btn-success btn-lg">Sign up</a>
```

I think that it would be better without the exclamation mark.

other than that, looks great. thanks for working on that!